src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMSource.java

```diff
@@ -178,9 +178,13 @@ public BitbucketSCMRepository getBitbucketSCMRepository() {
         return repository;
     }
 
-    CustomGitSCMSource getAndInitializeGitSCMSourceIfNull() {
+    CustomGitSCMSource getFullyInitializedGitSCMSource() {
```

Comment:
I had a go at renaming this method to make it more reflective of what it's actually doing